src/catalog/backends/common/src/main/java/org/geoserver/cloud/event/remote/resourcepool/RemoteEventResourcePoolProcessor.java

```diff
@@ -8,18 +8,16 @@
 
 import lombok.extern.slf4j.Slf4j;
 
-import org.geoserver.catalog.Catalog;
-import org.geoserver.catalog.CatalogInfo;
-import org.geoserver.catalog.ResourceInfo;
-import org.geoserver.catalog.ResourcePool;
+import org.geoserver.catalog.*;
```

Comment:
No wildcard imports please

Reply:
Done in 33c46d24e75dd4a50af3d908695a1e739d75ce4b